worksheet_D/oxo.py

```diff
@@ -0,0 +1,108 @@
+
+class OxoBoard:
+    def __init__(self):
+        #Initialises board array
+        self.gameBoard = [[0, 0, 0], [0, 0, 0], [0, 0, 0]]
```

Comment:
Neat, but could be made more expandable by using list multiplication and/or list comprehension